autoscale-scaler-marathon/pom.xml

```diff
@@ -60,6 +60,13 @@
             <artifactId>validation-api</artifactId>
             <scope>provided</scope>
         </dependency>
+
+        <!-- https://mvnrepository.com/artifact/org.apache.httpcomponents/httpclient -->
```

Comment:
Remove this comment - all open-source artifacts are in Maven Central (or The Central Repository as it calls itself these days).